index.html

```diff
@@ -39,19 +39,42 @@
         <h2>Controls</h2>
         <h3>Date Range</h3>
         <form name="dateSelection">
-          <div class="dateRadio">
+          <div class="radioBtn">
             <input type="radio" name="radioDate" id="radioMonth" checked>
             <label for="radioMonth">Past 30 Days</label>
           </div>
-          <div class="dateRadio">
+          <div class="radioBtn">
             <input type="radio" name="radioDate" id="radioWeek">
             <label for="radioWeek">Past 7 Days</label>
           </div>
-          <div class="dateRadio">
+          <div class="radioBtn">
             <input type="radio" name="radioDate" id="radioDay">
             <label for="radioDay">Today</label>
           </div>
         </form>
+        <h3>Data Filter</h3>
+        <form name="typeSelection">
+          <div class="checkBtn">
```

Comment:
Would also be neat to have a background color the same as the pin. Might need to have the `checkBtn` div add a little extra padding to make it work well